src/webview/components/memory-table.tsx

```diff
@@ -266,6 +272,14 @@ export class MemoryTable extends React.PureComponent<MemoryTableProps, MemoryTab
 
         return (
             <div className='flex-1 overflow-auto px-4'>
+                <Tooltip
+                    // On mouseover of a .hoverable element, this.handleOnBeforeTooltipShow runs and generates this.state.hoverContent
+                    target=".hoverable"
+                    onBeforeShow={this.handleOnBeforeTooltipShow}
+                    onHide={this.handleOnTooltipHide}
+                    showDelay={1000}
```

Comment:
I know I requested the delay but 1000 feels like too long now, maybe going down a bit might help? What do you think, @jreineckearm?

Reply:
I am fine going down a little with that. We should only make sure that the user has enough time without the tooltip when moving through the table in a stop and go manner, for example when using the mouse as a kind of means to guide their eye movement (not sure if that made sense).

Reply:
The default hover delay in the VSCode editor is 300ms. I'll use that.